### DISCUSSION TITLE
Diff view mysteriously switched from Split to Unified

### DISCUSSION THREAD

**Discussion by andrewrondeau-optirtc** [community/community#55636] [Pull Requests]
### Select Topic Area

Bug

### Body

I've always configured github to show the Files Changed tab in "split" mode.

This morning, it mysteriously switched back to "unified" without any intervention on my part.

Did a recent release / push / whatever forget my setting?

**Comment by plummercj** [comment]
I just had this happen and came across this old post. Turned out I had gotten logged out of git. Once i logged back in, split view was the default again.
